### ISSUE TITLE
Capitalize doesn't always work on slide text

### ISSUE THREAD

**Issue by joelgraff** [ChurchApps/FreeShow#753]
Capitalize on slide text doesn't always work.

The bug appears to be related to newly-pasted or typed text.  I believe selecting another slide, then selecting the target slide again avoids the bug.  However, applying the capitalize action on a slide with text that has been just added will not capitalize the start of the added text.

Tested on Windows with version 1.2.4.

**Comment by vassbo** [comment]
I tried different things and it worked each time. It's supposed to capitalize the first letter on each line, does it not?

**Comment by joelgraff** [comment]
Ha!  It's the Shift+Enter trick that's screwing it up.  I just hand-typed in some lines.  The ones I pressed Enter prior to fixing worked perfectly.  The ones I used Shift+Enter on, did not.

Maybe it would be best to just disable Shift+Enter altogether?  It doesn't appear to serve any real purpose and it would give users like me who are too smart for their own good a clue.  I only use it because hitting Enter often kicks you out of the control and sets the focus somewhere else - blame Microsoft for that, I guess.
